doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib

```diff
@@ -35,10 +35,64 @@ Sergio García Gómez (Telefónica I+D), Martin Bauer (NEC).
   
 ## Status
 
-This is a work in progress and is changing on a daily basis.
-Please send your comments to fiware-ngsi@lists.fiware.org.
-You can trace the discussions checking the archives of the mailing list:
-https://lists.fiware.org/private/fiware-ngsi/ (list subscription required).
+This specification is the final and stable version of NGSIv2 API specification (v2.0).
+
+# Changelog
+
+Changes since RC-2018.07:
+
+* None
+
+Changes since RC-2008.04:
+
+* Section "System/builtin Attributes" renamed to "Builtin Attributes"
+* Section "System/builtin Metadata" renamed to "Builtin Metadata"
+* Added new builtin attribute: `dateExpires`
+* Improved description in "Error Response" section
+* Added new option for `POST /v2/entities` operation: `upsert`
+* Clarifying about forwarding compliance in "Group Registrations"
+* `actionType` values for `POST /v2/op/update` operation are now in camelCase
+* Renamed field `attributes` to `attrs` in `POST /v2/op/query` operation
+* Added new field in `POST /v2/op/query` operation: `expression` (old `scope` field has been removed)
+* Added new operation `POST /v2/op/notify`
+
+Changes since RC-2017.11:
+
+* `orderBy` may include also `id` and `type` as ordering fields
+* Included `registrations_url` in `GET /v2` response
+* Added registration management operations
+   * `GET /v2/registrations`
+   * `POST /v2/registrations`
+   * `GET /v2/registrations/{id}`
+   * `PATCH /v2/registrations/{id}`
```

Comment:
Did we implement PATCH?   Sorry, out-of-memory ...

Reply:
No. It is part o the "Orion implementation gap", described at https://fiware-orion.readthedocs.io/en/master/user/ngsiv2_implementation_notes/index.html

Reply:
Ah, ok.
What is "implemented" is the description in the doc :)

NTC